relay_messages_test.go

```diff
@@ -43,8 +46,10 @@ const (
 )
 
 type testCallReqParams struct {
-	flags   byte
-	arg2Buf []byte
+	flags           byte
+	hasTChanThrift  bool
```

Comment:
can we make this `argScheme Format` and just pass in what we want (whether it's `Thrift` or `Raw`) and if nothing is set, then just not set the header. that way we can test for both cases (not set, and not "thrift") in the test cases

Reply:
done